docs/zh/cli-client/asset/issue-token.md

```diff
@@ -20,10 +20,10 @@ iriscli asset issue-token [flags]
 | --gateway          | string  | false    |               | 网关的唯一标识, 当 source 为 gateway 时必填                  |
 | --symbol           | string  | true     |               | Source内全局唯一的资产标识符，字母和数字的组合，首字符必须为字母，长度[3,8]，大小写无关 |
 | --canonical-symbol | string  | false    |               | 当 source 为 gateway 时，用来指定源链上的 Symbol，字母和数字的组合，长度[3,8]，大小写无关 |
```

Comment:
We don't enforce validation rule on `CanonicalSymbol` any more.

Reply:
Done